modules/apps/transfer/ibc_module_test.go

```diff
@@ -268,6 +273,103 @@ func (suite *TransferTestSuite) TestOnChanOpenAck() {
 	}
 }
 
+func (suite *TransferTestSuite) TestOnTimeoutPacket() {
+	var path *ibctesting.Path
+	var packet channeltypes.Packet
+
+	testCases := []struct {
+		name     string
+		malleate func()
+		expError error
+	}{
+		{
+			name:     "success",
+			malleate: func() {},
+			expError: nil,
+		},
+		{
+			name: "non-existent channel",
+			malleate: func() {
+				packet.SourceChannel = "channel-100"
+			},
+			expError: ibcerrors.ErrNotFound,
```

Comment:
You could remove the `name`, `malleate` and `expError` keys to be consistent with the two test cases below.